src/main/java/sparksoniq/jsoniq/runtime/iterator/functions/arrays/ArraySizeFunctionIterator.java

```diff
@@ -34,7 +34,11 @@
 public class ArraySizeFunctionIterator extends LocalFunctionCallIterator {
 
 
-    private RuntimeIterator arrayIterator;
+    /**
```

Comment:
Is this intentional?

Reply:
This was done automatically by Eclipse by using the bulk quick fix option on the missing serialization IDs. Given the very large number of files that need editing, I think the efforts required to manually remove these empty comments would outweigh the benefits. What do you think?